model/models.ts

```diff
@@ -208,23 +208,23 @@ let primitives = [
                  ];
 
 let enumsMap: {[index: string]: any} = {
-        "AccountDetailsAndActionsStatusEnum": AccountDetailsAndActionsStatusEnum,
-        "AccountStatusEnum": AccountStatusEnum,
-        "AddressTypeEnum": AddressTypeEnum,
-        "CategoriesEnum": CategoriesEnum,
-        "CategoryEnum": CategoryEnum,
-        "ClassificationEnum": ClassificationEnum,
-        "CountryEnum": CountryEnum,
-        "CreditNoteStatusEnum": CreditNoteStatusEnum,
-        "CurrencyEnum": CurrencyEnum,
-        "InvoiceTypeEnum": InvoiceTypeEnum,
-        "IssueStatusEnum": IssueStatusEnum,
-        "MethodEnum": MethodEnum,
-        "PurchaseOrderStatusEnum": PurchaseOrderStatusEnum,
-        "RequestFormatEnum": RequestFormatEnum,
-        "StateEnum": StateEnum,
-        "Status7d1Enum": Status7d1Enum,
-        "SyncStatusStatusEnum": SyncStatusStatusEnum,
+        "AccountDetailsAndActionsStatusEnum": AccountDetailsAndActionsStatusEnumExpected,
```

Comment:
Because the non-"Expected" name is now a type alias of the enum's expected values | string, this will fail unless we give the raw enum value which now has "Expected" suffix.